core/src/trezor/loop.py

```diff
@@ -184,7 +184,7 @@ def _step(task: Task, value: Any) -> None:
     this_task = task
     try:
         if isinstance(value, BaseException):
-            result = task.throw(value)  # type: ignore
+            result = task.throw(value)
             # error: Argument 1 to "throw" of "Coroutine" has incompatible type "Exception"; expected "Type[BaseException]"
             # rationale: In micropython, generator.throw() accepts the exception object directly.
```

Comment:
If we remove `type: ignore` should we remove also this comment? Because it is explaining the ignore isn't it?

Reply:
that's a good point, and I'm actually wondering _why_ the error went away

Reply:
maybe new version of mypy added implicit conversion from Exception to BaseException ?

Reply:
but the problem was that we're passing an _instance_ where `throw()` expects a _type_

i'll look into it

Reply:
```
Revealed type is 'Overload(
def (Type[builtins.BaseException], builtins.object =, Union[types.TracebackType, None] =) -> Any,
def (builtins.BaseException, None =, Union[types.TracebackType, None] =) -> Any
)'
```
:woman_shrugging: i'll take it i guess. maybe a change in Python's definition of `throw()`?

Reply:
welp! https://github.com/python/typeshed/issues/4241

Reply:
427b346

Reply:
Nice :)